account_invoice_line_sequence/models/account_move.py

```diff
@@ -9,7 +9,7 @@
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
-    sequence = fields.Integer(
+    sequence_show = fields.Integer(
```

Comment:
this is wrong, don't rename it. it's intended to rewrite the original sequence field.